Modifications

jonathon-love opened the pull request.

AlbertoAlvarezIglesias2019 committed:
I've added the coin package (for the logrank tests) and the survminer package (for extra plotting
functionality)

AlbertoAlvarezIglesias2019 committed:
Many changes here mainly bringing the code that I had used previously.
- The tables have been modified. There are three tables now. One with the event summaries, another one with the median estimates and another one with the results of the tests.
- The tests table includes a few variants of the log-rank test (as before) but now there is only one test (independent of the number of levels of the grouping variable).
- The plot of the survival function now includes the option of adding a risk table and the median.
- The plot of the cumulative hazard now includes the option of using the logarithmic transformation.
- Now it is possible to add the units (as open text) for the xaxis labels of the plots.
- I changed the order of the variables in the left panel (now the time variable is chosen first).
- I allowed the event indicator to be continuous or categorical.

AlbertoAlvarezIglesias2019 committed:
No input from me in this file. All the changes are automatic.

AlbertoAlvarezIglesias2019 committed:
- Event indicator can be now continuous.
- I removed the hf plot (this wasn't used anyway)
- I added the new options
* risk table
* plot the median
* use log scale for chf
* time units for the xaxis labels of the plot

AlbertoAlvarezIglesias2019 committed:
Changes to the layout of the tables. Adding the table with the estimation of the median survival times. Remove the plot for the hf.

AlbertoAlvarezIglesias2019 committed:
Minor changes and additions.

jonathon-love commented on jamovi/surv.a.yaml:
```diff
@@ -10,23 +10,24 @@ options:
+
     - name: event
-      title: Event
+      title: Event indicator (1/0)
```
Let's keep this as `Event` ... as we still allow the user to provide nominal variables

AlbertoAlvarezIglesias2019 replied:
I updated as suggested.

jonathon-love commented on jamovi/surv.a.yaml:
```diff
@@ -10,23 +10,24 @@ options:
       type: Data
 
+    - name: elapsed
+      title: Time
```
what do you think about 'Time Elapsed'? i have a slight preference for it, but don't mind if you prefer `Time`

AlbertoAlvarezIglesias2019 replied:
That's ok. I made the change.

jonathon-love commented on jamovi/surv.a.yaml:
```diff
@@ -71,4 +66,24 @@ options:
+      default: false
+      
+    - name: plotmedian
+      title: Add median 
```
how about?

```
  - name: median
    title: Median(s)
```

we try and avoid verbs in option names/titles

AlbertoAlvarezIglesias2019 replied:
Great to know :-) I updated the name and title.

jonathon-love commented on jamovi/surv.a.yaml:
```diff
@@ -71,4 +66,24 @@ options:
+      type: Bool
+      default: false
+      
+    - name: timeunits
```
`units`?

AlbertoAlvarezIglesias2019 replied:
I changed timeunits by units

jonathon-love commented on jamovi/surv.r.yaml:
```diff
@@ -4,8 +4,8 @@ title: Survival Analysis
-    - name: summary
-      title: Survival Analysis
+    - name:  compsurvTable1
+      title: Events Summary
```
why don't we try and keep the names of these tables similar to their titles? we want participants ... sorry, i've been working on experiments all day ... we want *users* to be able to, in RStudio, go:

```
results <- surv(...)
results$ ... <tab>
```

and from the tab list that RStudio suggests, we want them to be able to guess which item corresponds to the table they're interested in.

ideally, your changes would be backwards compatible too, but sometimes that's not possible.

AlbertoAlvarezIglesias2019 replied:
That's no problem (it makes a lot of sense !). I've changed it to the original name "summary". I've also changed compsurvTable3 by medianestimates.

jonathon-love commented on jamovi/surv.r.yaml:
```diff
@@ -18,24 +18,44 @@ items:
           visible: (groups)
-        - name: censored
-          title: Censored
-          type: integer
```
i could see a count of censored events being useful in some contexts ...

AlbertoAlvarezIglesias2019 replied:
I was in doubt as to keep it or not but I think you are right. It's probably better to include it as well. I made the change.

jonathon-love commented on jamovi/surv.r.yaml:
```diff
@@ -44,120 +64,86 @@ items:
-        - name: nuse[logrank]
-          title: SE
+        - name: chisqr[logrank]
+          title: Chi-Squared
```
χ² ?

AlbertoAlvarezIglesias2019 replied:
I made the change.

jonathon-love commented on jamovi/surv.r.yaml:
```diff
@@ -44,120 +64,86 @@ items:
-          title: z
-          format: zto,p
+        - name: df[logrank]
+          title: DF
```
the jamovi convention is `df`

AlbertoAlvarezIglesias2019 replied:
Good to know :-)

jonathon-love commented on jamovi/surv.r.yaml:
```diff
@@ -44,120 +64,86 @@ items:
-        - name: p[logrank]
-          title: p
+        - name: pvalue[logrank]
+          title: P-Value
```
the jamovi convention is simply `p`

AlbertoAlvarezIglesias2019 replied:
No problem. I made the change

jonathon-love commented on jamovi/surv.r.yaml:
```diff
@@ -44,120 +64,86 @@ items:
       visible: (sc)
       width: 600
-      height: 400
+      height: 600
```
i prefer the look of 600x400, but i'm happy with 600 high if you think it's better

AlbertoAlvarezIglesias2019 replied:
The reason I did this is because of the risk table (which requires space at the bottom). If there are many levels for the grouping variable the table runs out of space. Not sure if there is a way to change the height of the the plot dynamically (depending on whether the risk table option is checked) but my guess is that it wouldn't be easy. 
I'll keep it as it is for the moment.

jonathon-love replied:
inside `.init()` you can go:

```{r}
if (self$options$risktable)
    self$results$plot$setSize(600, 600)  # at least i think you can
```

AlbertoAlvarezIglesias2019 replied:
This is working now :-) Thanks for the help !

jonathon-love commented on jamovi/surv.a.yaml:
```diff
@@ -71,4 +66,24 @@ options:
       type: Bool
       default: false
+  
+    - name: risktable
```
just call this one `risk` ... we don't normally include `table` or `image` on the end of names

AlbertoAlvarezIglesias2019 replied:
Ok. I made the change

jonathon-love commented on R/surv.b.R:
```diff
@@ -40,150 +52,237 @@ survClass <- R6::R6Class(
         },
         .run = function() {
             
+            library(tidyr)
```
calls to `library()` are verboten!

AlbertoAlvarezIglesias2019 replied:
Ok :-) 
I actually didn't use it at the end anyway. I've removed that line.

jonathon-love commented on R/surv.b.R:
```diff
@@ -40,150 +52,237 @@ survClass <- R6::R6Class(
+                                       )
             
+            print(p)
             return(TRUE)
```
rather than printing `p` and returning `TRUE`, you can just return `p` ... i've been meaning to update documentation on this one. returning `p` is better.

AlbertoAlvarezIglesias2019 replied:
I made the change :-)

jonathon-love commented on R/surv.b.R:
```diff
@@ -40,150 +52,237 @@ survClass <- R6::R6Class(
+                    
+                }
+                    
+                } 
```
indentation not quite right here

AlbertoAlvarezIglesias2019 replied:
I fixed that

jonathon-love commented on R/surv.b.R:
```diff
@@ -40,150 +52,237 @@ survClass <- R6::R6Class(
+
+            conf.int <- self$options$ci
+            censYN <- self$options$cens
+            risktableYN <- self$options$risktable
```
i'd discourage the use of `.`s in variable names (in death watch i've used camelCase), and i'd discourage the `YN` suffix ... but i probably don't care that much in the end.

AlbertoAlvarezIglesias2019 replied:
Ok. I changed the names of the objects to 'ci', 'cens' and 'risk'

jonathon-love commented on R/surv.b.R:
```diff
@@ -40,150 +52,237 @@ survClass <- R6::R6Class(
+            
+            
+
+            
```
lot of space here

AlbertoAlvarezIglesias2019 replied:
Lol I tend to do that many times. It's like fresh air for me when programming :-) Too much air this time ! I've just removed it

jonathon-love commented on R/surv.b.R:
```diff
@@ -40,150 +52,237 @@ survClass <- R6::R6Class(
                     }
+                    
                 }
+                tt$setNote("note1","Log-Rank and Peto-Peto tests use survdiff from the survival package. Gehan and Tarone-Ware tests are obtained from the coin package.",init=TRUE)
```
are you better off using the reference system for this?

AlbertoAlvarezIglesias2019 replied:
I think I'm using the reference system now (just used the addFootnote method) and it looks much better.

jonathon-love replied:
oh, sorry, by reference system, i mean the little reference numbers which appear below tables.

(you possibly turned this feature off from the kebab menu, so you might have forgotten it was there)

so this is how we let people know that the ANOVA uses the car package:

![Screen Shot 2020-06-05 at 11 16 44](https://user-images.githubusercontent.com/3240247/83826012-33e93380-a71e-11ea-98f4-47120eeaf740.png)

you define your references in the `00refs.yaml` file:

https://github.com/jamovi/jmv/blob/master/jamovi/00refs.yaml

and then you attach the to results elements in the .r.yaml like so:

https://github.com/jamovi/jmv/blob/master/jamovi/ancova.r.yaml#L71-L75

(you can even attach them to individual columns. if the column/table isn't visible, then the reference won't be displayed).

and you can set them programmatically too:

https://github.com/jamovi/jmv/blob/master/R/ancova.b.R#L171-L175

jonathon-love commented on R/surv.b.R:
```diff
@@ -40,150 +52,237 @@ survClass <- R6::R6Class(
+                        exp=fit$exp[i]
+                        ))
+                }
+                }            
```
indentation

AlbertoAlvarezIglesias2019 replied:
I fix that

jonathon-love commented on R/surv.b.R:
```diff
@@ -40,150 +52,237 @@ survClass <- R6::R6Class(
+            
         },
-        .plot=function(image, theme, ggtheme, ...) {
+        .plot=function(image, ...) {
```
oh, can we honour the theme the user has selected?

at the very least we should set the background colour transparent.

AlbertoAlvarezIglesias2019 replied:
So that's what it is for?  Sorry I didn't know why it was there. I've put it back.

jonathon-love replied:
depending on the colour scheme selected from the kebab menu, `theme` and `ggtheme` will contain the different theme/colour scheme information.

you typically just go:

```
plot <- plot + ggtheme
```

and your plots will change colour scheme, to match what the user wants.

AlbertoAlvarezIglesias2019 replied:
I've updated this. I had some trouble because ggtheme overrides the legend options (I remove the legend if there is no grouping variable). Luckily this can be included in the ggsurvplot call :-)
So this is fixed now and it looks great !

jonathon-love reviewed (commented):
hey,

this is looking really great. here are some suggestions.

cheers

AlbertoAlvarezIglesias2019 committed:
- Changes in variable names and other object names based on jamovi conventions.
- Change the size of the plot based on risk table status.
- Use AddFootnote method in table instead of setNote.
- Add ggtheme and theme to the call of .plot
- Other minor changes

AlbertoAlvarezIglesias2019 committed:
Automatic updates

AlbertoAlvarezIglesias2019 committed:
Change the name and the title in some of the options.

AlbertoAlvarezIglesias2019 committed:
Change some names and title for some of the results
Add the number of censored observations to the summary table

AlbertoAlvarezIglesias2019 committed:
automatic changes

AlbertoAlvarezIglesias2019 committed:
Initial commit

AlbertoAlvarezIglesias2019 committed:
I removed the footnotes in the table as I'm using the reference system

AlbertoAlvarezIglesias2019 committed:
Automatic updates

AlbertoAlvarezIglesias2019 committed:
Inital comit

AlbertoAlvarezIglesias2019 committed:
Add the references in the columns of the table

AlbertoAlvarezIglesias2019 commented:
Hi Jonathon,
Thanks for sending the information (it's great to be able to add references 🙂 ). I've updated the code and committed the changes. Please let me know what you think.
Regards,
Alberto.
________________________________
From: Jonathon Love <notifications@github.com>
Sent: Friday 5 June 2020 02:23
To: jonathon-love/deathwatch <deathwatch@noreply.github.com>
Cc: Alvarez-Iglesias, Alberto <alberto.alvarez-iglesias@nuigalway.ie>; Comment <comment@noreply.github.com>
Subject: Re: [jonathon-love/deathwatch] Modifications (#5)


@jonathon-love commented on this pull request.

________________________________

In R/surv.b.R<https://github.com/jonathon-love/deathwatch/pull/5#discussion_r435640967>:

>                  }
+                tt$setNote("note1","Log-Rank and Peto-Peto tests use survdiff from the survival package. Gehan and Tarone-Ware tests are obtained from the coin package.",init=TRUE)


oh, sorry, by reference system, i mean the little reference numbers which appear below tables.

(you possibly turned this feature off from the kebab menu, so you might have forgotten it was there)

so this is how we let people know that the ANOVA uses the car package:

[Screen Shot 2020-06-05 at 11 16 44]<https://user-images.githubusercontent.com/3240247/83826012-33e93380-a71e-11ea-98f4-47120eeaf740.png>

you define your references in the 00refs.yaml file:

https://github.com/jamovi/jmv/blob/master/jamovi/00refs.yaml

and then you attach the to results elements in the .r.yaml like so:

https://github.com/jamovi/jmv/blob/master/jamovi/ancova.r.yaml#L71-L75

(you can even attach them to individual columns. if the column/table isn't visible, then the reference won't be displayed).

and you can set them programmatically too:

https://github.com/jamovi/jmv/blob/master/R/ancova.b.R#L171-L175

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/jonathon-love/deathwatch/pull/5#discussion_r435640967>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AN7DXE5JN3HCAGSWI5SULY3RVBCK7ANCNFSM4NSJ4ZYA>.

jonathon-love commented:
looking good!

sorry, one last little tweak:

1. disable the `Confidence intervals`, `Censored events`, etc. options when the `Survival curve` option isn't checked. you can do this by adding to each option, in the .a.yaml:

```
enable: (sc)
```

(where `sc` is the name of the survival curve option)

ditto with the `Log scale` under `Cumulative hazard function`.

this clarifies the relationship between the options, and prevents those situations where the user chooses 'Confidence intervals' under 'Plots' and wonders why nothing appears ... (which i just did :P)

pair-wise comparisons? people will want these (i assume), and we did have them. there's often disappointment when a useful feature is removed? we don't want to add them back in?

cheers and thanks for your efforts here! sorry i didn't do a proper review covering everything the first time.

AlbertoAlvarezIglesias2019 committed:
Add the reference for the EnvStats package

AlbertoAlvarezIglesias2019 committed:
Add back the option of pairwise comparisons

AlbertoAlvarezIglesias2019 committed:
Automatic updates

AlbertoAlvarezIglesias2019 committed:
Add the reference for the EnvStats package

AlbertoAlvarezIglesias2019 committed:
Add the option of including pairwise comparisons

AlbertoAlvarezIglesias2019 committed:
Add the table for pairwise comparisons in the results definitions.
A few other minor updates

AlbertoAlvarezIglesias2019 committed:
Automatic updates.

AlbertoAlvarezIglesias2019 committed:
Remove preformated text (I thought I had removed it)

AlbertoAlvarezIglesias2019 commented:
Hi Jonathon,
No problem about the updates. I've disabled the options you described conditioned on the state of sc and chf.
As for the pairwise comparisons I guess it is worth keeping it since it was there before (in case someone finds it useful). I fairness I've never seen that type of analysis for a logrank test (although I can see the logic of it). I've looked it up and there are a few papers that mention it but in my experience it's not very common. I've added it back anyway with some minor cosmetic updates. I used a corrected p-value (using Bonferroni) and added a note to make it clear. I also added the reference to the package you used to perform the test.
I've committed all the changes. Let me know how it looks now and no problem if you think it needs additional changes.
I tried to disable the option of multiple comparisons, as in enable: (option), whenever there are less than 3 levels for the grouping variable but I couldn't figure it out (it'd be great to have it like that as it wouldn't make sense to do pairwise comparisons unless there are 3 or more levels).
Regards,
Alberto.

________________________________
From: Jonathon Love <notifications@github.com>
Sent: Tuesday 9 June 2020 02:31
To: jonathon-love/deathwatch <deathwatch@noreply.github.com>
Cc: Alvarez-Iglesias, Alberto <alberto.alvarez-iglesias@nuigalway.ie>; Comment <comment@noreply.github.com>
Subject: Re: [jonathon-love/deathwatch] Modifications (#5)


looking good!

sorry, one last little tweak:

  1.  disable the Confidence intervals, Censored events, etc. options when the Survival curve option isn't checked. you can do this by adding to each option, in the .a.yaml:

enable: (sc)


(where sc is the name of the survival curve option)

ditto with the Log scale under Cumulative hazard function.

this clarifies the relationship between the options, and prevents those situations where the user chooses 'Confidence intervals' under 'Plots' and wonders why nothing appears ... (which i just did :P)

pair-wise comparisons? people will want these (i assume), and we did have them. there's often disappointment when a useful feature is removed? we don't want to add them back in?

cheers and thanks for your efforts here! sorry i didn't do a proper review covering everything the first time.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/jonathon-love/deathwatch/pull/5#issuecomment-640974594>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AN7DXE2RKQBQ43VJB2YE4ZLRVWGF3ANCNFSM4NSJ4ZYA>.

jonathon-love commented:
> I've committed all the changes

ooh, have you pushed your commits? it doesn't look like they're there.

> I tried to disable the option of multiple comparisons, as in enable: (option), whenever there are less than 3 levels for the grouping variable but I couldn't figure it out

why not just compare the two levels? sure, there's no point, but it's often easier to just do what the user has asked without worrying about the extra work.

alternatively, you could just provide an empty pairwise comparisons table with a footnote, or throw an error -- but i'd be inclined to just compare the two levels and call it a day :)

cheers

AlbertoAlvarezIglesias2019 commented:
Sorry I think I forgot to push the changes. I've done that now.
With only two levels it would be a duplicated analysis which I never like. At the moment you can't do it with less than 3 levels but I think its a good idea to through some message . I'll try to make the change today.
Regards,
Alberto.
________________________________
From: Jonathon Love <notifications@github.com>
Sent: Wednesday 10 June 2020 04:43
To: jonathon-love/deathwatch <deathwatch@noreply.github.com>
Cc: Alvarez-Iglesias, Alberto <alberto.alvarez-iglesias@nuigalway.ie>; Comment <comment@noreply.github.com>
Subject: Re: [jonathon-love/deathwatch] Modifications (#5)


I've committed all the changes

ooh, have you pushed your commits? it doesn't look like they're there.

I tried to disable the option of multiple comparisons, as in enable: (option), whenever there are less than 3 levels for the grouping variable but I couldn't figure it out

why not just compare the two levels? sure, there's no point, but it's often easier to just do what the user has asked without worrying about the extra work.

alternatively, you could just provide an empty pairwise comparisons table with a footnote, or throw an error -- but i'd be inclined to just compare the two levels and call it a day :)

cheers

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/jonathon-love/deathwatch/pull/5#issuecomment-641701942>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AN7DXE5KEBEVITJT522VVETRV36NJANCNFSM4NSJ4ZYA>.

AlbertoAlvarezIglesias2019 committed:
Allow the pairwise comparison table to appear with aa note if there are less than 3 levels.
Also I made the risk table to automatically change size depending on the number of levels of the grouping variable (otherwise it run out of space very quickly)

AlbertoAlvarezIglesias2019 committed:
Automatic updates

AlbertoAlvarezIglesias2019 commented:
Hi again Jonathon,
I've committed (and push) the changes to allow the pairwise comparison table to pop up with a note whenever there are less than 3 levels of the grouping variable. Le me know if you see anything else.
Regards,
Alberto.
________________________________
From: Jonathon Love <notifications@github.com>
Sent: Wednesday 10 June 2020 04:43
To: jonathon-love/deathwatch <deathwatch@noreply.github.com>
Cc: Alvarez-Iglesias, Alberto <alberto.alvarez-iglesias@nuigalway.ie>; Comment <comment@noreply.github.com>
Subject: Re: [jonathon-love/deathwatch] Modifications (#5)


I've committed all the changes

ooh, have you pushed your commits? it doesn't look like they're there.

I tried to disable the option of multiple comparisons, as in enable: (option), whenever there are less than 3 levels for the grouping variable but I couldn't figure it out

why not just compare the two levels? sure, there's no point, but it's often easier to just do what the user has asked without worrying about the extra work.

alternatively, you could just provide an empty pairwise comparisons table with a footnote, or throw an error -- but i'd be inclined to just compare the two levels and call it a day :)

cheers

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/jonathon-love/deathwatch/pull/5#issuecomment-641701942>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AN7DXE5KEBEVITJT522VVETRV36NJANCNFSM4NSJ4ZYA>.

jonathon-love commented:
haha, i'm so eager to merge this, one *last* thing :P

we normally put the levels in the pairwise comparisons in separate columns, like this:

```
#
#  Pairwise comparisons - len                
#  ───────────────────────────────────────── 
#                    W           p           
#  ───────────────────────────────────────── 
#    500     1000    6.373279    0.0000197   
#    500     2000    7.596746    0.0000002   
#    1000    2000    5.321635    0.0004929   
#  ───────────────────────────────────────── 
# 
#
```

among other things, it makes it easier for people using the R package to consume the results (they don't need to parse the cells in the table).

cheers

AlbertoAlvarezIglesias2019 committed:
Going back to showing the pairs in two separate columns (for the pairwise comparison)

AlbertoAlvarezIglesias2019 committed:
Automatic updates

AlbertoAlvarezIglesias2019 committed:
Going back to showing the pairs in two separate columns (for the pairwise comparison)

AlbertoAlvarezIglesias2019 commented:
Ok Jonathon, no problem. I reversed the change to its original state. I committed and push the changes.
Can't wait to see it all together as well 🙂
Regards,
Alberto.
________________________________
From: Jonathon Love <notifications@github.com>
Sent: Thursday 11 June 2020 02:10
To: jonathon-love/deathwatch <deathwatch@noreply.github.com>
Cc: Alvarez-Iglesias, Alberto <alberto.alvarez-iglesias@nuigalway.ie>; Comment <comment@noreply.github.com>
Subject: Re: [jonathon-love/deathwatch] Modifications (#5)


haha, i'm so eager to merge this, one last thing :P

we normally put the levels in the pairwise comparisons in separate columns, like this:

#
#  Pairwise comparisons - len
#  ─────────────────────────────────────────
#                    W           p
#  ─────────────────────────────────────────
#    500     1000    6.373279    0.0000197
#    500     2000    7.596746    0.0000002
#    1000    2000    5.321635    0.0004929
#  ─────────────────────────────────────────
#
#


among other things, it makes it easier for people using the R package to consume the results (they don't need to parse the cells in the table).

cheers

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/jonathon-love/deathwatch/pull/5#issuecomment-642346073>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AN7DXE5X2364QN5MLYZNJRLRWAVH7ANCNFSM4NSJ4ZYA>.

jonathon-love commented:
oh man, we are so close :P

this is what my paired comparisons table looks like:

```
#
#  Pairwise Comparisons 
#  ──────────────────── 
#                   
#  ──────────────────── 
#    500     1000   
#    500     2000   
#    1000    2000   
#  ──────────────────── 
# 
#
```

one other thing, `Paired comparisons` rather than `Paired Comparisons` (in the options)

cheers

AlbertoAlvarezIglesias2019 commented:
Hi Jonathon,
I can't find the typo (Compairsons). I checked in github as well and can't find it either. What file did you see it in?
As for the the output you showed, is not that right? This is what it looks like for me for the logrank.

[cid:59e106dd-bc4c-4fab-bb80-f6c83d746c3c]
________________________________
From: Jonathon Love <notifications@github.com>
Sent: Friday 12 June 2020 08:58
To: jonathon-love/deathwatch <deathwatch@noreply.github.com>
Cc: Alvarez-Iglesias, Alberto <alberto.alvarez-iglesias@nuigalway.ie>; Comment <comment@noreply.github.com>
Subject: Re: [jonathon-love/deathwatch] Modifications (#5)


oh man, we are so close :P

this is what my paired comparisons table looks like:

#

#  Pairwise Comparisons

#  ────────────────────

#

#  ────────────────────

#    500     1000

#    500     2000

#    1000    2000

#  ────────────────────

#

#



one other thing, Paired comparisons rather than Paired Compairsons

cheers

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/jonathon-love/deathwatch/pull/5#issuecomment-643131849>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AN7DXE3WCRVVCPORALKU3YTRWHN3JANCNFSM4NSJ4ZYA>.

jonathon-love commented:
here:

https://github.com/AlbertoAlvarezIglesias2019/deathwatch/blob/master/jamovi/surv.a.yaml#L51

ah! the pairwise comparisons are only performed when you've got some tests selected - i can't think of a way to make that clearer. sorry, my bad. (incidentally, if you include an image in a response to a github email, the image gets stripped).

cheers

jonathon-love commented:
sorry, it's not a typo, it's that we use Sentence case for options, not Title Case.

jonathon-love commented:
i'll merge this now, and fix it as i merge.

AlbertoAlvarezIglesias2019 commented:
I tried to enable the pairwise comparisons condition based on selection of at least one of the test but didn't work 🙁 It only works if you use one of the tests but of course that is of no use anyway ...
________________________________
From: Jonathon Love <notifications@github.com>
Sent: Friday 12 June 2020 12:17
To: jonathon-love/deathwatch <deathwatch@noreply.github.com>
Cc: Alvarez-Iglesias, Alberto <alberto.alvarez-iglesias@nuigalway.ie>; Comment <comment@noreply.github.com>
Subject: Re: [jonathon-love/deathwatch] Modifications (#5)


here:

https://github.com/AlbertoAlvarezIglesias2019/deathwatch/blob/master/jamovi/surv.a.yaml#L51

ah! the pairwise comparisons are only performed when you've got some tests selected - i can't think of a way to make that clearer. sorry, my bad. (incidentally, if you include an image in a response to a github email, the image gets stripped).

cheers

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/jonathon-love/deathwatch/pull/5#issuecomment-643217424>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AN7DXE4BLIHDR6YHAO4MXGLRWIFF3ANCNFSM4NSJ4ZYA>.

AlbertoAlvarezIglesias2019 commented:
What should I do now Jonathon? Do I pull the updates?
Regards,
Alberto.
________________________________
From: Jonathon Love <notifications@github.com>
Sent: Friday 12 June 2020 12:21
To: jonathon-love/deathwatch <deathwatch@noreply.github.com>
Cc: Alvarez-Iglesias, Alberto <alberto.alvarez-iglesias@nuigalway.ie>; Comment <comment@noreply.github.com>
Subject: Re: [jonathon-love/deathwatch] Modifications (#5)


Merged #5<https://github.com/jonathon-love/deathwatch/pull/5> into master.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/jonathon-love/deathwatch/pull/5#event-3438013681>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AN7DXE5J6LT6A6EJTQDHIRTRWIFS5ANCNFSM4NSJ4ZYA>.

jonathon-love commented:
so this is where git gets a bit messy.

what i've done is squashed all your commits into one (i do this to keep the git history easy to follow). so our two trees will now have diverged. so you want to roll back all your commits:

`git reset HEAD~30 --hard`

i chose 30, but any large number will do, provided it's more than the number of commits you made, but less than the total number of commits.

then you'll rebase of of mine:

```
git remote add upstream https://github.com/jonathon-love/deathwatch.git
git fetch upstream
git rebase upstream/master
```

at this point, your repo will be identical to mine ... however, it will only be the repo on your computer, and not the one on gh ... so you need to push to gh *however* because you've rewritten your git history, you'll need to force push

```
git push --force
```

you can do all that ... *or* you can just delete your repo, and clone mine over again :P
